libraries/pod/src/slice.rs

```diff
@@ -14,6 +14,7 @@ use {
 
 const LENGTH_SIZE: usize = std::mem::size_of::<PodU32>();
 /// Special type for using a slice of `Pod`s in a zero-copy way
+#[derive(Debug)]
```

Comment:
This get left in? 👀

Reply:
Required for `.expect_err()`, which was added to the test below. 🤷‍♂️

Reply:
Ohh that's weird. I hate to say it, but I'd rather see you use `unwrap_err` like you did here than add this trait:

https://github.com/solana-labs/solana-program-library/pull/5119/commits/da7e6e5fba7e302f60c7f412279b4fcb76a8411a#diff-715e96a717478b935eac0729cd96d09beaaa5426483ce87d791e82a882bd8f48R251-R262

Reply:
Oh that's silly. It will let you do `.err().unwrap()` though, but not `unwrap_err()`.

> No worries, I know these things matter.

Yeah, I just would hate to add extra code/methods just for tests, but `Debug` isn't the worst thing I guess

Reply:
Updated, please check.

Reply:
Just make `rustfmt` happy and I'm STAMPIN' it

Reply:
My bad, fixed now.